### DISCUSSION TITLE
Internal blogging platform?

### DISCUSSION THREAD

**Discussion by elianiva** [wrideveloper/wridev.id#18] [Ideas]
Should we also make the website as a blogging platform? We do have a medium already, but it seems like it's going to be easier for people to contribute if we have our own platform since we have the full control on how to publish it and how it would look like.

Also, I am planning to finish the website before Hacktoberfest so that we can utilise this feature to encourage people to contribute at least an article for their Hacktoberfest token instead of just contributing a random hello world text file.

The format should be as simple as a plain markdown since it is used everywhere and can be learned in less than an hour. No need for an external API call which should make it fast.

wdyt?

**Comment by alizul01** [comment]
i think it's a good idea but lets make the wri-dev website MVP first and work as expected, then as long as we go we develop the blog system?
